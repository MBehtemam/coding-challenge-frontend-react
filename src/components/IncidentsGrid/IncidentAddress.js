import styled from 'styled-components';

const IncidentAddress = styled.address`
  font-style: italic;
`;

export default IncidentAddress;
