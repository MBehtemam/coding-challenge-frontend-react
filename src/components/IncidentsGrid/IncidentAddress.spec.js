import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import IncidentAddress from './IncidentAddress';

describe('Test suits for Incidetn Address', () => {
  it('should match with snapshot', () => {
    const tree = renderer.create(<IncidentAddress>210 Berlin</IncidentAddress>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
