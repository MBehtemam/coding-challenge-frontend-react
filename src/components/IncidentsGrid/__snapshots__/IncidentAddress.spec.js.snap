// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Incidetn Address should match with snapshot 1`] = `
.c0 {
  font-style: italic;
}

<address
  className="c0"
>
  210 Berlin
</address>
`;
